Source/Resources/Bridge.js

```diff
@@ -579,4 +579,22 @@ function wrappingSentencesWithinPTags(){
     }
     
     guessSenetences();
+}
+
+// Class based onClick listener
+
+function addClassBasedOnClickListener(schemeName, className, parameterName) {
+	// Get all elements with the given className
+	var elements = document.getElementsByClassName(className);
```

Comment:
I was thinking about it, maybe we could use the JavaScript `querySelector` or `querySelectorAll`, instead of just `getElementsByClassName`. The `querySelector` is like jQuery, so we can do like `#someID`, `.someclass`, `h1` or any kind of complex selection.